openapi/components/schemas/Common/CommonTransaction.yaml

```
@@ -87,7 +88,10 @@ properties:
     allOf:
       - $ref: ../CurrencyCode.yaml
   requestAmount:
-    description: Amount of the payment request. If an adjustment occurs, the purchased amount may differ from the billing amount.
+    description: |-
+      Amount of the payment request.
+      If an adjustment occurs,
+      the purchased amount may differ from the billing amount.
```

```suggestion
      the purchase amount may differ from the billing amount.
```
I think this means literally the `purchaseAmount` field above.